Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1
+
+ADD . /project
+WORKDIR /project
+
+RUN pip install -r requirements.txt
+
+CMD python src/manage.py runserver 0.0.0.0:8000
```

Comment:
Yes, but by convention it's a good idea to always add it to the Dockerfile, becuase that way it's self-describing.